lib/sycamore/sycamore/docset.py

```diff
@@ -549,13 +549,15 @@ def extract_graph_structure(self, extractors: list[GraphExtractor], **kwargs) ->
                     .explode()
                 )
         """
-        from sycamore.transforms.extract_graph import ExtractDocumentStructure
+        from sycamore.transforms.extract_graph import ResolveEntities, ExtractDocumentStructure
 
-        self.plan = ExtractDocumentStructure(self.plan)
         docset = self
+
+        docset.plan = ExtractDocumentStructure(docset.plan)
         for extractor in extractors:
             docset = extractor.extract(docset)
-
+        if len(extractors) > 0:
```

Comment:
nit but maybe just stick the for loop under this too so all extractors stuff is under

Reply:
fixed